examples/2016-10-31/usage_plan/template.yaml

```diff
@@ -0,0 +1,98 @@
+Conditions:
+  C1:
+    Fn::Equals:
+      - true
+      - true
+
+Parameters:
+  UsagePlanType:
+    Type: String
+    Default: SINGLE
+
+Globals:
+    Api:
+      OpenApiVersion: 3.0.0
+      Auth:
+        ApiKeyRequired: true
+        UsagePlan:
+          CreateUsagePlan: !Ref UsagePlanType
+
+Resources:
+   MyApiOne:
+     Condition: C1
+     Type: AWS::Serverless::Api
+     Properties:
+       StageName: Prod
+
+   MyApiTwo:
+     Type: AWS::Serverless::Api
+     Properties:
+       StageName: Prod
+       Auth:
+         ApiKeyRequired: true
```

Comment:
you shouldnt need this to be defined here again if its in globals

Reply:
Will update the template